server/client_manager.py

```diff
@@ -1048,17 +1048,21 @@ def send_area_list(self, full=False):
                 msg += f"[{area.id}] {area.name} {users}{status}{owner}{hidden}{locked}{pathlocked}{passworded}{muted}"
             self.send_ooc(msg)
 
-        def get_area_info(self, area_id, mods, afk_check):
+        def get_area_info(self, area_id, mods, afk_check, hub_id = -1):
             """
             Get information about a specific area.
             :param area_id: area ID
+            :param hub_id: hub ID
             :param mods: limit player list to mods
             :param afk_check: Limit player list to afks
             :returns: information as a string
             """
             info = "\r\n"
             try:
-                area = self.area.area_manager.get_area_by_id(area_id)
+                if hub_id == -1:
+                    area = self.area.area_manager.get_area_by_id(area_id)
+                else:
+                    area = self.server.hub_manager.hubs[hub_id].areas[area_id]
```

Comment:
These changes don't seem to relate to the PR

Reply:
It is necessary to modify get_area_info, otherwise we'll have:

 area = self.area.area_manager.get_area_by_id(area_id)
 
 And so it will check only the areas of the hub where the client is

Reply:
ohhh okay